Update "Installing and Updating" Category (#66)
dema-trading-ai/docs: docs/installation_and_updating/updating.md

---
hide:
  - toc
---

# Updating

> The Engine gets regular updates. It is wise to always make sure you are running the newest version of the Engine. With

Reviewer: ```suggestion
The Engine gets regular updates. It is wise to always make sure you are running the newest version of the Engine. With
```
The Engine is updated regularly. For optimal performance make sure you are running the newest version of the Engine.

Author: ```suggestion
The Engine is updated regularly. For optimal performance make sure you are running the newest version of the Engine.
```
